File: 백준/Silver/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def GCD(x,y):
    while y:
        x, y = y, x % y
    return x

def LCM(x,y):
    result = (x*y)//GCD(x,y)
    return result
x,y = map(int,input().split())

print(GCD(x,y))
print(LCM(x,y))

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.kt
import java.util.*

fun gcd(a: Int, b: Int): Int {
    return if (b == 0) a else gcd(b, a % b)
}

fun lcm(a: Int, b: Int): Int {
    return a * b / gcd(a, b)
}

fun main() {
    val scanner = Scanner(System.`in`)
    val a = scanner.nextInt()
    val b = scanner.nextInt()

    println(gcd(a, b))
    println(lcm(a, b))
}